coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -141,3 +141,16 @@ def _parse_documentation_with_symbols(self, param_identifiers,
                 parsed.append(self.Description(desc=desc))
 
         return parsed
+
+    def assemble(self):
+        """
+        Assembles a list of parsed documentation comments.
+
+        This function assembles the whole documentation comment,
+        with the given markers and indentation.
+        """
+        lines = self.documentation.splitlines(keepends=True)
+        assembled = self.indent + self.marker[0]
+        assembled += ''.join('\n' if line == '\n' else self.indent +
```

Comment:
I think you need to add to the if: `and not self.marker[1]`, as only if there's no middle-marker you can leave the line completely empty :)

e.g.
```python
# hello
#
# world
```